docs/providers/aws/guide/functions.md

```diff
@@ -484,3 +486,18 @@ functions:
       onSuccess: otherFunctionInService
       onFailure: arn:aws:sns:us-east-1:xxxx:some-topic-name
 ```
+
+### Maximum Event Age and Maximum Retry Attempts for Asynchronous Invocations
+
+When intention is to invoke function asynchronously you may want to configure `maximumRetryAttempts` and/or `maximumEventAge` for it.
```

Comment:
Sentence _When intention is.._ is duplicated among sections, therfore maybe organize this as follows:

```md
## Asynchronous invocation 

When intention is to invoke function asynchronously you may want to configure following additional settings

### Destinations

[destination targets](https://docs.aws.amazon.com/lambda/latest/dg/invocation-async.html#invocation-async-destinations) can be the other...

...

### Maximum Event Age and Maximum Retry Attempts

`maximumEventAge` accepts values between 60
...
```